src/components/Incidents/IncidentsChart/IncidentsChart.jsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+
+export function IncidentGanttChart({ groups, viewport }) {
+    // TODO useResizeObserver()
+    const width = 550;
+    const height = 100;
+
+    return (
+      <React.Component
```

Comment:
maybe this can be just a `<div>` with the specific styling. But this might be just an initial approach and I'll leave up to you for further PRs